src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  box-sizing: border-box;
+  color: $main-blue;
+  display: flex;
+  flex-direction: column;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  border: $border-card solid #f3f3f3;
+  margin: 0;
+  border-radius: 5px;
+  width: $card-width;
+  font-family: "Roboto", sans-serif;
+  @include Transition(transform);
+  &__image {
+    display: flex;
+    height: 134px;
+    width: 160px;
+    margin: 32px auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin: 0 16px 4px;
+
+  }
+
+  &__product-code {
+    color: $main-gray;
+    margin: 0 16px;
+  }
+
+  &__review {
+    // width: 166px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 16px 16px 24px;
+    align-items: center;
+    // gap: 17px;
+  }
+
+  &__reviews {
+    margin: 0;
+  }
+
+  &__priceBlock {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 16px;
+  }
+
+  &__price {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: $main-gray;
+    margin: 0;
+  }
+
+  &__value {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    margin: 0;
+  }
+
+  &__link {
+    box-sizing: border-box;
+    margin: 16px;
+    height: 40px;
+    width: 166px;
```

Comment:
Consider avoiding the hardcoded width property, and make it more flexible: use min-width instead